releasenotes/notes/deprecate-mitigation-f5f6ef3233b3d726.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations_misc:
+  - |
+    The ``qiskit.result.mitigation`` module has been deprecated and will be removed in the 2.0 release.
+    The deprecation includes the ``LocalReadoutMitigator`` and ``CorrelatedReadoutMitigator`` classes 
```

Comment:
You didn't want to use `class` refs for these? They seem to be in the docs: https://docs.quantum.ibm.com/api/qiskit/qiskit.result.LocalReadoutMitigator

Reply:
I didn't remember if we would use refs for removals, I will keep it in mind and suggest the changes in the reno copy-editing phase.